File: a1/src/main/java/project/a1/dto/a1/ZahtevZaAutorskaDelaDTO.java
package project.a1.dto.a1;

import lombok.Getter;
import lombok.Setter;
import project.a1.dto.main_schema.TLiceDTO;
import project.a1.model.a1.AutorskoDelo;
import project.a1.model.a1.Prilozi;
import project.a1.model.a1.TAutor;
import project.a1.model.main_schema.TLice;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.datatype.XMLGregorianCalendar;
import java.util.Date;

@XmlRootElement
public class ZahtevZaAutorskaDelaDTO {
    public TLiceDTO podnosilacPrijave;
    public TLiceDTO punomocnik;
    public AutorskoDeloDTO autorskoDelo;
    public PriloziDTO prilozi;
    public TAutorDTO autor;
    public Date datumPodnosenja = new Date();
    public Integer broj = 3;
    public Boolean podnosilacJeAutor;

}

File: XML_client/src/app/a1/model/zahtev-za-autorska-dela-d-t-o.ts
import {TLiceDTO} from "../../shared-models/t-lice-d-t-o";
import {AutorskoDeloDTO} from "./autorsko-delo-d-t-o";
import {TAutor} from "./t-autor";
import {Prilozi} from "./prilozi";

export class ZahtevZaAutorskaDelaDTO {

  constructor(
    public broj:number = 11,
    public datumPodnosenja: string = '',
    public podnosilacPrijave:TLiceDTO = new TLiceDTO(),
    public punomocnik:TLiceDTO = new TLiceDTO(),
    public autorskoDelo: AutorskoDeloDTO = new AutorskoDeloDTO(),
    public autor: TAutor = new TAutor(),
    public prilozi: Prilozi = new Prilozi(),
    public podnosilacJeAutor: boolean = false
  ) {}

}
